src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.test.jsx

```diff
@@ -56,6 +63,8 @@ describe('LegacyConfigForm', () => {
 
   afterEach(() => {
     axiosMock.reset();
+    defaultAppConfig.discussionTopics = [{ name: 'General', id: 'course' },
+      { name: 'Edx', id: '13f106c6-6735-4e84-b097-0456cff55960' }];
```

Comment:
any reason to do this in `afterEach` rather than `beforeEach`?

Reply:
just wanted to reset discussionTopics which I am altering during the test case running. It can be done both ways before running as well. but I thought as we are resetting like cleaning up the test case effect in afterEach, it's also more like a cleanup task hence I did it in afterEach.

Reply:
I don't think you need to alter these in your tests? What is the use case here?

Reply:
when we add two topics in a test case as we are not passing index and calculating it using default topics list length the length remains the same and both topics pick the same index for typing topic name.